modin/pandas/test/test_general.py

```diff
@@ -690,6 +690,8 @@ def test_to_pandas_indices():
 
     pd_df = md_df._to_pandas()
 
+    df_equals(md_df.index, md_df._query_compiler.to_pandas().index)
```

Comment:
This check doesn't test our case because md_df is not empty. We should add a parameter named `data`, which would have two values (`test_data_values[0]` and `[]`, to this test. After that you can remove this line and leave the body of the test as it is.